web/site/js/vis/ActionBar.js

```diff
@@ -230,6 +233,10 @@ export default class ActionBar extends VObject {
         }
         this.vis.actionBar.refresh()
     }
+    
+    actionOpenEnergyWindow() {
+        window.open(document.location.href.replace("/visualize","/chartsViewer"), '_blank');
```

Comment:
Initially, I thought of creating a container inside the simulator window. I actually started trying to integrate `charts.js` into `PIXI.js`, but the way they are meant to be used made it very complicated to me.

It can be probably done by someone with more experience in both PIXI.js and HTML/Javascript itself, but apparently the use of container by PIXI made it hard to connect with canvas used by chart.js.

At some point, I even thought of making charts myself inside PIXI environment, but this would be a rework of all the feature already available with `charts.js`. Also, these charts could end up polluting the view of nodes/network.

Adding a secondary tab allow us to create a complete new page, without interfering the network view, nor performance. It allows us to see the charts in another screen too, if necessary.